composer.json

```diff
@@ -34,6 +34,7 @@
         "drupal/entity_reference_display": "^1.2",
         "drupal/entity_reference_revisions": "^1.3",
         "drupal/facets": "^1.0",
+        "drupal/facets_pretty_paths": "^1.0",
```

Comment:
pytanko czy potrzebne nam to jako globalne dependency dla Drooplera, czy może powinno być podobnie jak `commerce` zewnętrzną zależnością, wpisaną jako wymagana w `d_commerce_product_list.info.yml`?

Reply:
W pliku info od modułu jest podana ta zależność, tylko pytanie skąd automagicznie moduł ten zostanie dociągniety? Czy nie będzie i ktoś będzie musiał sam sobie dociągnać?

Reply:
jeśli wygra opcja, że nie dodajemy modułów obowiązkowo do drooplera - to user dostanie do łapy instrukcję co ma dociągnąć